src/components/common/App/AppInput.vue

```diff
@@ -0,0 +1,36 @@
+<script>
+import { defineComponent } from 'vue';
+export default defineComponent({
+  props: {
+    placeholder: {
+      type: String,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      inputContent: '',
```

Comment:
So, right now this input will in fact hold it's own data but - it will not pass it anywhere, so using it will be quite limited, especially in the context of forms.
Is it a work in progress and something like a `v-model` implementation will be added in? 😄

Reply:
The reality is actually much worse 😉 The simplest version of v-model on input tag works here, as long as my component is a simple input with no wrappers or additional elements, so I was happy with this quick result here. But now I see I can't go far with this lazy solution, especially when it comes to building forms, so I will definitely learn more about `update:modelValue` thing and apply it here and in AppSelect soon.